README.md

```diff
@@ -14,14 +14,22 @@ Zadania są dostępne na stronie [opanujfrontend.pl/advent](https://opanujfronte
 
 Każdego dnia wykonaj w repozytorium polecenie `npm run create` a następnie skopiuj kod ze strony.
 
+Użyj komendy `npm run create all`, jeśli chcesz wygenerować foldery z plikami startowymi
```

Comment:
I think it's best to reference explicit names of npm scripts just to be consistent. 

`npm run create:month` as suggested by @MarcinParda
`npm run create:?` - I can't find any single-word name for "up until today"